### DISCUSSION TITLE
✋hand pointer function

### DISCUSSION THREAD

**Discussion by Mano2731995** [pymupdf/PyMuPDF#2096] [Looking for help] [answered]
Hai jorj how are you? 

Here I have attached code and output 

In main pdf i have attached  pdf file(1811 xyz. Pdf)  in str(1811) by using file attachment annot. 
My requirement is when we move over to string(1811) need handpointer instead of text cursor. 
How can i do that? 

![IMG_20221130_191731.jpg](https://user-images.githubusercontent.com/117537492/204824681-ec6b08a7-b482-47a3-aca1-851cc5d7d109.jpg)
[]()![IMG_20221130_191909.jpg](https://user-images.githubusercontent.com/117537492/204824506-31466ba5-2e98-4ee1-8c8a-385db8fbcc6d.jpg)

**Comment by JorjMcKie** [comment, marked as answer]
The mouse symbol is under control of your PDF viewer. Adobe displays something else than another viewer ...

**Reply by Mano2731995** [reply]
Ok jorj. By using page.insertlink(see image last line) we can able to get hand symbol when move over to string. 
If any possible to merge the file attachment annot and page. Insertlink

Page.insertlink only for creating dummy link for hand symbol

**Reply by JorjMcKie** [reply]
>  If any possible to merge the file attachment annot and page. Insertlink

No, will not work.

**Reply by Mano2731995** [reply]
> >  If any possible to merge the file attachment annot and page. Insertlink
> 
> No, will not work.

Ok jorj thank you
